paratest/node/src/service.rs

```diff
@@ -217,11 +219,10 @@ async fn start_node_impl<Network: sc_network::NetworkBackend<Block, Hash>>(
         let client = client.clone();
         let transaction_pool = transaction_pool.clone();
 
-        Box::new(move |deny_unsafe, _| {
+        Box::new(move |_| {
```

Comment:
Is something changed also in the way we specify if to allow/disallow unsafe RPC methods when running an RPC node ?

Reply:
Yes; `deny_unsafe` is specific to an rpc interface now (and there can be many different interfaces).
The CLI switch to configure custom interfaces is `--experimental-rpc-endpoint`